homeassistant/components/ezviz/sensor.py

```diff
@@ -18,27 +18,55 @@
 PARALLEL_UPDATES = 1
 
 SENSOR_TYPES: dict[str, SensorEntityDescription] = {
-    "sw_version": SensorEntityDescription(key="sw_version"),
```

Comment:
It seems like you are removing a sensor which is a breaking change! Please do this in a separate PR!

Reply:
Hi Joostlek,

That key doesn't exist in the api so it never creates a sensor. Has no effect so the reson for removing.

Reply:
The dictionary returned is defined here: https://github.com/BaQs/pyEzviz/blob/e693a82cb8437dc72fb41697ae3e000803b11e05/pyezviz/camera.py#L93

The actual key is called "version" and I think it was renamed a very long time ago and never updated in the sensor.

Reply:
Right, I'll approve it, but I want someone else to check it aswell to make sure this is a good call :)